integration_tests/integration/make_a_referral/referralSectionVerifier.js

```diff
@@ -0,0 +1,65 @@
+// eslint-disable-next-line max-classes-per-file
+export default class ReferralSectionVerifier {
+  static get verifySection() {
+    return new _ReferralSectionChecker()
+  }
+}
+function hrefAttrChainer(isActive) {
+  return isActive ? 'have.attr' : 'not.have.attr'
+}
+class _ReferralSectionChecker {
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
I wonder if we should turn this rule off, I don't think it's that useful a rule - what do you think?

Reply:
Sorry, I'm not sure what you mean?

This rule is required to ensure that the linter passes. It complains because "This" isn't used within the class method

Reply:
I meant, should we turn the rule off in eslint config? I don't think it's a very useful linting rule that we should enforce.

Reply:
In this case the rule is necessary to avoid linting failures.

It fails because `this` is not used within the method; it's probably suggesting that the method should be static.

Instead what I've done now is increased the chaining so that these methods return `this` so no need to add these off-rules anymore.